hw/ip/spi_device/doc/_index.md

```diff
@@ -49,6 +69,38 @@ The SW can receive TPM commands with payload (address and data) and respond to t
 The submodule provides the command, address, write, and read FIFOs for the SW to communicate with the TPM host system.
 The submodule also supports the SW by managing a certain set of the FIFO registers and returning the read request by HW quickly.
 
+In Flash mode, SPI Device HWIP behaves as a Serial Flash device by recognizing SPI Flash commands and processing those commands by HW.
+The commands processed by HW are Read Status (1, 2, 3), Read JEDEC ID, Read SFDP, EN4B/ EX4B, and read commands with aids of SW.
+The IP supports Normal Read, Fast Read, Fast Read Dual Output, Fast Read Quad Output.
+This version of IP does not support Dual IO, Quad IO, QPI commands.
+
+In Passthrough mode, SPI Device receives SPI transactions from a host system and forwards the transactions to a downstream flash device.
+SW may filter prohibited commands by configuring 256-bit {{< regref "FILTER" >}} CSR.
+The IP cancels ongoing transaction if the received opcode matches to the filter CSR by de-asserting CSb and gating SCK  to the downstream flash device.
+
+SW may program CSRs to change the address and/or the first 4 bytes of payload on-the-fly in Passthrough mode.
+The address translation feature allows SW to maintain A/B binary images without aids of the host system.
+The payload translation may be used to change the payload of Write Status commands to not allow certain fiels to be modified.
+
+Flash mode and Passthrough mode may share the datapath.
```

Comment:
Would it be more accurate to say that the flash and passthrough modes "can be active" at the same time?
Because data path sharing can also mean that the different operations are performend on the same blocks, and I am not sure this is the case here (I may be wrong, though).